README.md

```diff
@@ -53,63 +52,36 @@ nwbfile = NWBFile(
     session_start_time=datetime.datetime.now(datetime.timezone.utc),
 )
 
-# In the next ten calls or so, we'll set up the metadata from the bottom of the metadata tree up
-# You can follow along here: 
-
-# Create a commanded voltage container, this can store one or more commanded voltage series
-multi_commanded_voltage = MultiCommandedVoltage()
-
-# Add a commanded voltage series to this container
-commandedvoltage_series = (
-    multi_commanded_voltage.create_commanded_voltage_series(
-        name="commanded_voltage",
-        data=[1.0, 2.0, 3.0],
-        frequency=30.0,
-        power=500.0,
-        rate=30.0,
-        unit="volts",
-    )
-)
-
-# Create an excitation sources table
-excitationsources_table = ExcitationSourcesTable(
-    description="excitation sources table"
+# Create a Fibers table, and add one (or many) fiber
+fibers_table = FibersTable(description="fibers table")
+fibers_table.add_row(
+    location="my location",
```

Comment:
Looking at the Dombeck data, they are also storing the `depth` of each fiber.  
Would it make sense to also have `depth` as optional attribute in `FibersTable`?

Reply:
The lerner lab specifies the full bregma coordinates for each optical fiber (AP, ML, DV). I think including these coordinates would be the most complete way to specify fiber location. Does the Dombek paper mention coordinates for their fiber implants?

Reply:
I agree that the coordinates would be the most complete solution. Custom descriptions such as that could go on `location` or `description` I believe

Reply:
> Looking at the Dombeck data, they are recording from 1 or 2 fibers from various depths. (e.g. they start from 3.5 mm, record for 15 minutes then move down the fiber to 4.5 mm) I'm wondering how to represent this information.

Ah, that's interesting. Maybe an entry in the fibers table for each depth? Anyway, this PR discussion is already pretty long/bloated, so I think we should wrap this one up and then make dedicated PRs for smaller improvements (see #12)